state-chain/traits/src/lib.rs

```diff
@@ -1061,6 +1061,7 @@ pub trait SwapLimitsProvider {
 	fn validate_broker_fees(
 		broker_fees: &Beneficiaries<Self::AccountId>,
 	) -> Result<(), DispatchError>;
+	fn get_minimum_fee_for_broker(broker_id: &Self::AccountId) -> BasisPoints;
```

Comment:
```suggestion
	fn get_minimum_vault_swap_fee_for_broker(broker_id: &Self::AccountId) -> BasisPoints;
```

Since it shouldn't be applied to deposit channels, we should make this explicit in the name.

Reply:
Also, a nit, but `SwapLimitsProvider` is not really a good name for this trait any more since we've slowly been adding various other bits and pieces.

Ether we rename this to something more general, eg. `SwapValidationParameters`, or we break it up into more specific traits.